src/main/java/it/gov/pagopa/pu/bff/model/ServiceStatus.java

```diff
@@ -0,0 +1,16 @@
+package it.gov.pagopa.pu.bff.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class ServiceStatus {
```

Comment:
don't use the same DTO to get configuration and status, otherwise you will expose on the internet the internal urls
```suggestion
public class CoreServiceHealthStatusConfig {
```